jenkins-build-pack-api/Jenkinsfile

```diff
@@ -78,7 +78,7 @@ node() {
 	
     stage('Checkout code base') {
 		if (params.domain  && params.domain.trim() != "" ) {			
-			if(domain != "platform"){
+			if(domain != "platform" && configLoader.SCM.TYPE != "gitlab"){
```

Comment:
Why is this required? We ideally shouldn't have the diff between bitbucket and gitlab, so please comment above this line why is that required?

Reply:
This is because the diversity in the repo naming of platform services and user services.
In Gitlab the same webhook trigger job is using for platform services and user services.
So this if block is needed.

Reply:
I am not sure I quite follow. I believe this is a bug that needs to be fixed elsewhere (the deviation between how gitlabs jobs are triggered vs bitbucket jobs are triggered). Can you point those differences?